### PR TITLE
Update assetlist.json and Images Terra SEAS DAO

### PR DESCRIPTION
**Pull request by sk3ngy**
SEAS DAO token information

Please also add image to - terra2/images
 
SEAS

https://github.com/cosmos/chain-registry/assets/143271799/cb127a94-c82c-4e94-84d2-a76a6eae910b

### CODE REVIEW

**Comment by sk3ngy** [comment]
If anything is wrong please advise this is my first time

File: terra2/assetlist.json [hunk 1]
```diff
@@ -486,6 +486,28 @@
...
+            "denom": "seas",
+            "exponent": 6
+          }
+        ],
+        "base": "cw20:terra1rc6ssp5rym7a0hg29xpj4cc9e67tl56kg5jyzgl9qrhfxxc2ugvsnrkala",
+        "name": "Sailing The SeaS DAO",
+        "display": "seas",
+        "symbol": "SEAS",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra2/images/SEAS.png"
```
Comment (JeremyParish69) [inline terra2/assetlist.json:509]: this image doesn't exist?

Comment (JeremyParish69) [reply terra2/assetlist.json:509]: Please upload the image into that terra2/images/ folder in your branch

Comment (sk3ngy) [reply terra2/assetlist.json:509]: SEAS

I cannot could you do it  please ? link provided above

Comment (sk3ngy) [reply terra2/assetlist.json:509]: I get an error uploads are disabled .

Comment (sk3ngy) [reply terra2/assetlist.json:509]: File uploads require push access to this repository

Comment (JeremyParish69) [reply terra2/assetlist.json:509]: your branch is on your fork, so you should be able to

Comment (sk3ngy) [reply terra2/assetlist.json:509]: Nope access denied 
image

Comment (JeremyParish69) [reply terra2/assetlist.json:509]: Which repo and branch is that where uploads are disabled?

**Comment by sk3ngy** [comment]
Hey,

Cosmo/chain-registry/terra2/images

Thank you

File: terra2/assetlist.json (hunk 1 above)
Comment (sk3ngy) [reply terra2/assetlist.json:509]: Cosmo/chain-registry/terra2/images

Thank You

Comment (JeremyParish69) [reply terra2/assetlist.json:509]: That's the wrong location, as you don't have push permissions to the cosmos' repo. You should be trying to upload it to your forked repo, which you have full permissions over.
https://github.com/sk3ngy/chain-registry/tree/patch-1/terra2/images

Comment (sk3ngy) [reply terra2/assetlist.json:509]: All done :) Image upload successful thank you for your help.

Comment (JeremyParish69) [reply terra2/assetlist.json:509]: It's not there. Did you Commit after upload?

**Comment by sk3ngy** [comment]
Yes, just committed add files via upload #3161 On 9 Nov 2023, at 17:29, JeremyParish69 ***@***.***> wrote:﻿
@JeremyParish69 commented on this pull request.



In terra2/assetlist.json:
> +        "denom_units": [
+          {
+            "denom": "cw20:terra1rc6ssp5rym7a0hg29xpj4cc9e67tl56kg5jyzgl9qrhfxxc2ugvsnrkala",
+            "exponent": 0
+          },
+          {
+            "denom": "seas",
+            "exponent": 6
+          }
+        ],
+        "base": "cw20:terra1rc6ssp5rym7a0hg29xpj4cc9e67tl56kg5jyzgl9qrhfxxc2ugvsnrkala",
+        "name": "Sailing The SeaS DAO",
+        "display": "seas",
+        "symbol": "SEAS",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra2/images/SEAS.png"

It's not there. Did you Commit after upload?

File: terra2/assetlist.json (hunk 1 above)
Comment (JeremyParish69) [reply terra2/assetlist.json:509]: Can you add it to your own patch-1 branch so it's added in this same PR?

Comment (sk3ngy) [reply terra2/assetlist.json:509]: Done Thank You

**Pull request merged**